### PR TITLE
Game pages

### PR DESCRIPTION
**Pull request by DotSea**
1. Add Game page
2. refactor: rename and restructure component 'questionA'/'questionB' to 'taskA'/'taskB'
3. unify the question logic
- The question logic is consist of three parts: stage, question, task.
- Stage is the order of the question, the browser will show only ONE stage at the same time.
- Question is the combination of the task, which contain two task: 'taskA' and 'taskB'.
- Task is the main thing we want to ask: land usage (TaskA) and whether have illegal factory on certain land (TaskB).
- Tutorial page have 4 stages in total, each stage contain half of the question, which equals to 'taskA' OR 'taskB'.
- Game page have 5 stages in total, each stage contain one question, which equals to 'taskA' AND 'taskB'.

### CODE REVIEW

File: src/components/GameContent.vue [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    },
+    hasIllegalFactory() {
+      const data = JSON.parse(localStorage.getItem('SpotDiffData'));
+      return data[this.whichQuestion - 1].userAnswer.hasIllegalFactory;
+    },
+  },
+  props: { whichQuestion: Number, goToNextStage: Function },
+  async created() {
+    try {
+      await axios.patch('http://localhost:3000/db', {
```
Comment (Yukaii) [inline src/components/GameContent.vue:99]: @DotSea 記得這邊要換成 process.env.API_SERVER_URL + '/db'

Comment (DotSea) [reply src/components/GameContent.vue:99]: 好的了解。
也想請問一下，我需要創一個.env檔，裡面放mock database的url嗎？如果需要的話，是要創.env或是.env.local呢？

Comment (Yukaii) [reply src/components/GameContent.vue:99]: 都可以，有加到 .gitignore 就好~~

**Review by DotSea — COMMENTED** [review]
@Yukaii 這邊加好囉，因為vue cli只取 `VUE_APP_` 開頭的變數作為環境變數，所以我改成為`process.env.VUE_APP_SPOTDIFF_API_URL`。
如果netlify有相關的設定，再麻煩yukai囉

**Pull request merged**